crates/sui-graphql-client/src/query_types/object.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::query_types::{schema, Base64, PageInfo, SuiAddress, Uint53};
+
+// ===========================================================================
+// Object(s) Queries
+// ===========================================================================
+
+#[derive(cynic::QueryFragment, Debug)]
+#[cynic(schema = "rpc", graphql_type = "Query", variables = "ObjectQueryArgs")]
+pub struct ObjectQuery {
+    #[arguments(address: $address, version: $version)]
+    pub object: Option<Object>,
+}
+
+#[derive(cynic::QueryFragment, Debug)]
+#[cynic(schema = "rpc", graphql_type = "Query", variables = "ObjectsQueryArgs")]
+pub struct ObjectsQuery {
+    #[arguments(after: $after, before: $before, filter: $filter, first: $first, last: $last)]
+    pub objects: ObjectConnection,
+}
+
+// ===========================================================================
+// Object(s) Query Args
+// ===========================================================================
+
+#[derive(cynic::QueryVariables, Debug)]
+pub struct ObjectQueryArgs {
+    pub address: SuiAddress,
+    pub version: Option<Uint53>,
+}
+
+#[derive(cynic::QueryVariables, Debug)]
+pub struct ObjectsQueryArgs<'a> {
+    pub after: Option<&'a str>,
+    pub before: Option<&'a str>,
+    pub filter: Option<ObjectFilter<'a>>,
+    pub first: Option<i32>,
+    pub last: Option<i32>,
+}
```

Comment:
Are these types not already defined by the generated schema?

Reply:
I might misunderstand things, but I thought they are generated and only used when `build` is called on a query to verify this query against the generated schema. Not sure though, I might be totally off here but had the same question as you, and that's how I understood it needs to be done from the documentation.